deployment/terraform/engine.go

```diff
@@ -65,6 +63,14 @@ func (t *Terraform) runAWSCommand(ctx context.Context, args []string, dst io.Wri
 	return _runCommand(cmd, dst)
 }
 
+type cmdLogger struct {
+}
+
+func (*cmdLogger) Write(in []byte) (int, error) {
+	mlog.Info(string(in))
```

Comment:
Removes the newline at the end of the terraform output.

```suggestion
	mlog.Info(strings.TrimSpace(string(in)))
```

Reply:
Nice one